hue-cli.js

```diff
@@ -245,12 +245,20 @@ switch (args[0]) {
   case 'register': // register this app
     client = getclient();
     console.log('please go and press the link button on your base station');
-    client.register(function(err) {
+    client.register(function(err, resp) {
       if (err) {
         console.error('failed to pair to Hue Base Station %s', config.host);
         throw err;
       }
+      
       console.log('Hue Base Station paired!')
+      console.log('username: ' + resp[0].success.username);
+      config.username = resp[0].success.username;
+      
+      // writing config file
+      var s = JSON.stringify(config, null, 2);
```

Comment:
I didn't realize `hue-cli` had as large of a community as it did (thanks everyone!) so I'm interested in gathering insight from everyone.  My only concern with this is, when a user runs `hue register` it will clobber their existing config file without passing the `--save` option.  Admittedly, the `--save` function currently operates in a less-than-ideal way.

My current proposal would be to drop the `--save` functionality completely, and make it so `hue register` will automatically write out the config IF ONE DOES NOT ALREADY EXIST.  If one exists, `hue register` should print a warning, dump the current config, and exit non-zero, and leave it up to the user to remove it before registering a new bridge.  What do you all think of that?

Either way though, I'm currently 👍 on this change and appreciate the small diff!